src/main/java/org/broadinstitute/hellbender/cmdline/argumentcollections/OptionalReadInputArgumentCollection.java

```diff
@@ -13,10 +14,14 @@
 public final class OptionalReadInputArgumentCollection extends ReadInputArgumentCollection {
 
     @Argument(fullName = StandardArgumentDefinitions.INPUT_LONG_NAME, shortName = StandardArgumentDefinitions.INPUT_SHORT_NAME, doc = "BAM/SAM/CRAM file containing reads", optional = true)
-    public List<File> readFiles;
+    public List<String> readFilesNames;
```

Comment:
Ugh, we really need a generic URI/resource abstraction (no need to address in this branch, however).